packages/ocaml-glpk/ocaml-glpk.0.1.7/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "1.2"
+maintainer: "smimram@users.sourceforge.net"
+author: "Samuel Mimram"
+homepage: "http://ocaml-glpk.sourceforge.net/"
+dev-repo: "https://github.com/smimram/ocaml-glpk"
+bug-reports: "https://github.com/smimram/ocaml-glpk/issues"
+license: "GNU GPL v. 2"
+
+build: [
+  [make "byte"]
+  [make "opt"]
+]
+install: [
+  [make "install"]
+]
+remove: [["ocamlfind" "remove" "glpk"]]
+depends: ["ocamlfind"]
+depexts: [
+  [["debian"] ["libglpk-dev" >= "4.52.1"]]
```

Comment:
unfortunately this syntax is not supported in depexts. I'm fine to merge the PR if you just remove your last commit.